assets/onboarding/log-event.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Send log event.
+ *
+ * @param {string} eventName Event name.
+ * @param {Array} properties Event properties.
+ */
+export const logEvent = ( eventName, properties ) => {
+	window.sensei_log_event( eventName, properties );
+};
+
+/**
+ * Enable or disable event logging.
+ *
+ * @param {boolean} enabled Enabled state.
+ */
+logEvent.enable = ( enabled ) => {
+	window.sensei_event_logging.enabled = enabled;
+};
+
+/**
+ * Send log event when link is opened.
+ *
+ * @param {string} eventName Event name.
+ * @param {Array} properties Event properties.
+ * @return {Object} Element attributes.
+ */
+export const logLink = ( eventName, properties ) => {
+	const isMiddleButtonEvent = ( e ) => 1 === e.button;
+	return {
+		onClick: () => logEvent( eventName, properties ),
+		onAuxClick: ( e ) =>
```

Comment:
Cool! I didn't know this event. But I [read that it's focused in `non-primary pointing device button`](https://developer.mozilla.org/en-US/docs/Web/API/Element/auxclick_event). Is it what you wanted?

And I was wondering if set directly the `logEvent` in the link or button would be cleaner. Did you consider this possibility?

Reply:
Yes, this is for handling opening the links with the middle button - the onClick isn't triggered for that. 
I added the helper with the spread because this adds multiple attributes and it's easier to use.
`<a href="/" { ...logLink( 'event' ) }>` vs. `<a href="/" onClick={logLink( 'event')} onAuxClick={logLinkIfMiddleClick('event') } >`